includes/class-bitcoin-donation-shortcode-multi-amount-wide.php

```diff
@@ -10,18 +10,40 @@ public function __construct()
         add_shortcode('multi_amount_donation_wide', [$this, 'bitcoin_donation_multi_render_shortcode_wide']);
     }
 
+    private function get_template($template_name, $args = [])
+    {
+        if ($args && is_array($args)) {
+            extract($args);
+        }
+
+        $template = plugin_dir_path(__FILE__) . '../templates/' . $template_name . '.php';
+
+        if (file_exists($template)) {
+            include $template;
+        }
+    }
+
     function bitcoin_donation_multi_render_shortcode_wide()
```

Comment:
Did we agree to remove the `wide` shortcode and have it as a dropdown selector for all of the forms?